### ISSUE TITLE
Next@10.0.2 breaks builds that rely on @babel/runtime being resolved relatively.

### ISSUE THREAD

**Issue by RossMcMillan92** [vercel/next.js#19448]
# Bug report

## Describe the bug

Next has changed the way it imports `@babel/runtime`

**Next@10.0.1**
```js
var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
```

**Next@10.0.2**
```js
var _interopRequireDefault = require("/home/USER/PROJECT/node_modules/@babel/runtime/helpers/interopRequireDefault");
```

This has broken our deployments, as we build on one image and deploy on a different one, which would require a different absolute path.

We now get MODULE_NOT_FOUND errors. 

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. This happens by default on all Next.js builds

## Expected behavior

`@babel/runtime` is required with a relative path, so we don't have to worry about external environment.

## System information

- OS: Ubuntu (probably all OSes)
- Browser (if applies) [e.g. chrome, safari]
- Version of Next.js: 10.0.2
- Version of Node.js: 12

**Comment by Timer** [comment]
Closing this as fixed by #19542

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
